less/images.less

```diff
@@ -75,7 +75,15 @@
         }
       }
     }
-
+    .dropdown-menu > li > div.radio {
+      display: block;
+      padding: 3px 20px;
+      clear: both;
+      font-weight: normal;
+      line-height: 1.42857143;
```

Comment:
This comes from a copy paste from css in sibling element ( .dropdown-menu > li > a ).